algorithms/CPlusPlus/Dynamic-Programming/rod-cutting.cpp

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Please remove this file

Reply:
Hello Ming, I had a question, you asked me to remove this file what if i deleted this files from here ?
Will it get deleted every

Reply:
My bad, I saw two algorithm on this pull request.